addons/aircraft/CfgAmmo.hpp

```diff
@@ -39,15 +39,53 @@ class CfgAmmo {
 
     // also adjust tracer, "muh lightshow"; also adjust splash damage radius
     class Gatling_30mm_HE_Plane_CAS_01_F: BulletBase {
-        hit = 80; // default: 180
+        hit = 70; // default: 180
         indirectHit = 12; // default: 4
-        indirectHitRange = 3; // default: 3
+        indirectHitRange = 4; // default: 3
         caliber = 1.4; // default: 5
         deflecting = 3; // default: 5
         fuseDistance = 3; // default: 10
         tracerStartTime = 0.02; // default: 0.1
         timeToLive = 40; // default: 6
     };
+   
+    // adjust damage and splash damage, AP Rounds
+    class ACE_Gatling_30mm_AP_Plane_CAS_01_F: Gatling_30mm_HE_Plane_CAS_01_F {
+       explosive = 0.05; // default: 0.4
+       caliber = 5; // default: 5
+       deflecting = 8; // default: 5
+       hit = 110; // default: 180
+       indirectHit = 2.5; // default: 4
+       indirectHitRange = 1; // default: 3
+       fuseDistance = 0.2; // default: 10
+    };
+
+   // adds submunition logic, enabling multiple rounds per frame
+   class ACE_Gatling_30mm_Sub_HEI: SubmunitionBullet {
+       submunitionAmmo = "Gatling_30mm_HE_Plane_CAS_01_F";
+       weaponType = "cannon";
+       submunitionConeType[] = {"poissondisccenter", 3};
+       submunitionConeAngle = 0.056; // in degrees, 0.055 ~= 0.001 mils minute, but present
+       model = "\A3\Weapons_f\Data\bullettracer\tracer_red.p3d";
+       triggerTime = 0.005;
+       ACE_caliber = 1.4;
+   };
+
+   class ACE_Gatling_30mm_Sub_AP: ACE_Gatling_30mm_Sub_HEI {
+        submunitionAmmo = "ACE_Gatling_30mm_AP_Plane_CAS_01_F";
+        ACE_caliber = 1.4;
+   };
+   
+    class ACE_Gatling_30mm_Sub_CM41: ACE_Gatling_30mm_Sub_HEI {
+        submunitionAmmo[] = {"ACE_Gatling_30mm_AP_Plane_CAS_01_F",0.8,"Cannon_30mm_HE_Plane_CAS_01_F",0.2};
```

Comment:
RPT now throws error
```
21:43:13 No class Cannon_30mm_HE_Plane_CAS_01_F
21:43:13 No ammo class Cannon_30mm_HE_Plane_CAS_01_F
21:43:13 No class Cannon_30mm_HE_Plane_CAS_01_F
21:43:13 No ammo class Cannon_30mm_HE_Plane_CAS_01_F
```

I think this should be 
`Gatling_30mm_HE_Plane_CAS_01_F`
instead of 
`Cannon_30mm_HE_Plane_CAS_01_F`